packages/perseus-editor/src/editor-page.tsx

```diff
@@ -166,14 +166,7 @@ class EditorPage extends React.Component<Props, State> {
                 },
                 reviewMode: true,
                 legacyPerseusLint: this.itemEditor.current?.getSaveWarnings(),
-            }).extend(
-                _(this.props).pick(
-                    "workAreaSelector",
-                    "solutionAreaSelector",
-                    "hintsAreaSelector",
```

Comment:
We don't do anything with these _inside_ Perseus anymore and our consumers don't pass these in anymore.